### PR TITLE
Remove references to VEDA

### PR DESCRIPTION
**Pull request by emileten**
- I think it would be good that what's in `/data` reflects what's pushed to the database. So, I am trying to remove legacy VEDA json files that we're not planning to move. This is a first step, I removed step function input files that had references to veda S3 resources, and removed corresponding collection files. I think there are more things to remove. In addition, I am surprised there are a couple missing files. For example, we have some PALSAR data in our [dev catalog that doesn't have corresponding json files here.](https://stac-browser.maap-project.org/collections/Global_PALSAR2_PALSAR_Mosiac).
- Swapped 'VEDA' or 'veda' in the code for 'MAAP' or 'maap'
- removed outdated legacy issue templates.

### CODE REVIEW

File: OPERATING.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
-# Operating Guide: Data Transformation and Ingest for VEDA
+# Operating Guide: Data Transformation and Ingest for MAAP
 
-This guide provides information on how VEDA runs data ingest, transformation and metadata (STAC) publication workflows via AWS Services, such as step functions.
+This guide provides information on how MAAP runs data ingest, transformation and metadata (STAC) publication workflows via AWS Services, such as step functions.
 
 NOTE: Since collection ingest still requires calling the database from a local machine, users must add their IP to an inbound rule on the security group attached to the RDS instance.
```
Comment (jjfrench) [inline OPERATING.md]: We change change this section now that we're using the `/collections` endpoint

**Comment by jjfrench** [comment]
There are a handful of collections you don't see json's that are in the feature branches of the veda-data-pipelines repo. (We had published the collections for the "must-have datasets")

**Review by jjfrench — APPROVED** [review]

File: OPERATING.md (hunk 1 above)
Comment (emileten) [reply OPERATING.md]: Thanks @jjfrench, just removed the note.

**Comment by emileten** [comment]
> There are a handful of collections you don't see json's that are in the feature branches of the veda-data-pipelines repo. (We had published the collections for the "must-have datasets")

ah ok thanks ! i browsed through the branches to see if anything was crucial but didn't see it. I could push these branches to branches in this repo or let you guys add these manually.

**Comment by emileten** [comment]
This version was deployed to test and dev, and in both I published successfully with the poetry scripts the collections and items of the ESA CCI biomass 2018 dataset.

**Pull request merged**